README.md

```diff
@@ -28,3 +28,6 @@ Other:
 * `0 0 * * * bash /root/mpc-autofill/update_database >> /root/db_update.txt 2>&1`
 * `0 0 * * SUN bash /root/mpc-autofill/sync_dfcs`
 8. Deploy two Google Script according to the code specified in `autofill.py` and adjust the URLs in that script to point to your GS endpoints
+
+## Using Docker
```

Comment:
let's move this to the top of the readme since i imagine most people will use docker to run the project

Reply:
I completely restructured the main README and merged the one from the docker sub-folder. Now, the first part is about preparative steps (setting up Google Server Account, etc.) and the second part is about the technical installation. Have a look at the rendered README at https://github.com/fklemme/mpc-autofill. I left the manual installation part (which is now at the end) to PR #33.

Reply:
looks good! let's add a note for windows installation though. PR #33 might want to rebase onto this branch?

Reply:
Yeah, I think that's the plan. Let's get this PR merged. Then I'll use bits and pieces from my verbose README to flesh out this one.